packages/app-rfi/src/components/stepper/RfiStepperButtons.js

```diff
@@ -83,7 +83,7 @@ export const RfiStepperButtons = ({
               <Button
                 type="submit"
                 className="rfi-submit btn btn-primary"
-                disabled={!!isSubmitting}
+                disabled={submitCount > 0}
```

Comment:
SubmitCount is increased each time the form tries to submit or go to the next page. One page variant click submit and the form will be stuck 
<img width="522" alt="image" src="https://github.com/user-attachments/assets/22547772-07d4-4616-8cac-9613541f02e0">

Reply:
PR now uses the `totalSteps` in the disabled logic to account for steps getting logged as submits.

Reply:
Needs more work: Paging back and forth through the form can exceed the submit count.